packages/@uppy/utils/src/findDOMElement.ts

```diff
@@ -4,9 +4,9 @@ import isDOMElement from './isDOMElement.ts'
  * Find a DOM element.
  */
 export default function findDOMElement(
-  element: Node | string,
+  element: HTMLElement | string,
   context = document,
-): Node | null {
+): HTMLElement | null {
```

Comment:
Why this change? This will make code using e.g. `.parentNode` report errors, when it's perfectly fine